Makefile

```diff
@@ -95,9 +104,9 @@ build/%.asc:
 
 .PHONY: install release_builds release
 
-test: test/unit
+test: test/unit ## Run the tests
```

Comment:
WDYT about follow the same standard of kubebuilder and replace the "/" by "-"?

Reply:
@camilamacedo86 I think I'd rather make that a separate PR.